tests/cursor/bug0920-001.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+PHPC-924: Using a projection with an empty field name causes a crash when destroying cursor
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+$query = new MongoDB\Driver\Query([], ['projection' => ['' => 1]]);
+
+echo throws(function() use ($manager, $query) {
+    $cursor = $manager->executeQuery(NS, $query);
```

Comment:
It depends a little bit how much code it involves. I'm generally sceptical about doing these in two places. In general, I do think it's better to have an InvalidArgumentException here.